CONTRIBUTING.md

```diff
@@ -83,13 +101,50 @@ npm run dev:pristine
 npm run dev:format
 ```
 
+## Coding conventions
+
+1. We recommend small, single-purpose components. We recommend breaking up large components into smaller, more cohesive components. Smaller components are easier to test because you can isolate functionality to ensure that components are immune to changes in other components. In general, larger components lead to larger tests that are more difficult to write and more difficult to understand.
```

Comment:
In the docs site, the section where we are pulling this content from (https://developer.blackbaud.com/skyux/contribute/contribution-process/guidelines#components) has a second paragraph with more detailed guidance about naming and placement. Should we include any of that info here as well?

Reply:
I decided to avoid it as I felt it was a little much and the basic information there is easily figured out when working in our code base. And if not - we would catch it in code review.